src/load

```diff
@@ -607,8 +688,15 @@ def main():
 
     validate_opts(args_dict)
 
-    # now we need to fetch the data from NVD/NIST/MITRE using the API and populate the DB
-    for data in args.data:
+    # now we need to fetch the data from NVD/NIST/MITRE/Cyentia using the API and populate the DB
+    data_to_fetch = args.data.copy()
+    # Fetch data for 'cve' if both 'cve' and 'epss' are present
+    if 'cve' in data_to_fetch and 'epss' in data_to_fetch:
```

Comment:
just make sure to order the elements so that `cve` would be processed first and add `epss` to the list in case it was not specified but `cve` was specified, and leave the data load to be done by the `fetch_data` method.